riscv/encoding.h

```diff
@@ -4,7 +4,7 @@
 
 /*
  * This file is auto-generated by running 'make' in
- * https://github.com/riscv/riscv-opcodes (a014979)
+ * https://github.com/riscv/riscv-opcodes (0a1a87a)
```

Comment:
This hash is for a fixup commit on riscv-opcodes, so this should not be merged like this. Let's get that PR merged first and make sure this hash still refers to an actual commit.

Reply:
I think we need to update the `overlap_list.h` to declare this overlap.

Reply:
Fixed